src/components/ModalForm.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
```

Comment:
Because this is very specific, I think maybe it will be a good idea to build a model component that takes in children components and renders them. This way you will have a model for every occasion

Reply:
That's a great idea, thanks Cam!